azext_edge/edge/_help.py

```diff
@@ -970,40 +935,34 @@ def load_iotops_help():
         examples:
         - name: Add an event to an asset.
           text: >
-            az iot ops asset event add --asset {asset} -g {resource_group} --event-notifier {event_notifier}
+            az iot ops asset event add --asset myasset -g myresourcegroup --event-notifier eventId --name eventName
 
-        - name: Add an event to an asset with capability id, event name, observability mode, custom queue size,
+        - name: Add an event to an asset with event name, observability mode, custom queue size,
                 and custom sampling interval.
-          text: >
-            az iot ops asset event add --asset {asset} -g {resource_group} --event-notifier {event_notifier}
-            --name {name} --capability-id {capability_id} --observability-mode
-            {observability_mode} --queue-size {queue_size} --sampling-interval {sampling_interval}
-
-        - name: Add an event to an asset with the given pre-filled values.
           text: >
             az iot ops asset event add --asset MyAsset -g MyRG --event-notifier eventId --name eventName
-            --capability-id tagId1 --observability-mode log --queue-size 2 --sampling-interval 500
+            --observability-mode log --queue-size 2 --sampling-interval 500
     """
 
     helps[
         "iot ops asset event export"
     ] = """
         type: command
         short-summary: Export events in an asset.
-        long-summary: The file name will be {asset_name}_dataPoints.{file_type}.
+        long-summary: The file name will be {asset_name}_events.{file_type}.
         examples:
         - name: Export all events in an asset in JSON format.
           text: >
-            az iot ops asset event export --asset {asset} -g {resource_group}
+            az iot ops asset event export --asset myasset -g myresourcegroup
         - name: Export all events in an asset in CSV format in a specific output directory.
           text: >
-            az iot ops asset event export --asset {asset} -g {resource_group} --format csv --output-dir {output_directory}
+            az iot ops asset event export --asset myasset -g myresourcegroup --format csv --output-dir myAssetFiles
         - name: Export all events in an asset in CSV format that can be uploaded to the DOE portal.
           text: >
-            az iot ops asset event export --asset {asset} -g {resource_group} --format portal-csv
+            az iot ops asset event export --asset myasset -g myresourcegroup --format portal-csv
```

Comment:
May not be an issue, but using "portal" may conflate Azure Portal and the Digital Operations Experience.

Reply:
I might just update portal-csv to just be csv and then DOE -> Digital Operations Experience